test/common_e2e.dart

```diff
@@ -92,30 +91,7 @@ Future withAuthClient(List<String> scopes, AuthCallback callback,
         .clientViaServiceAccount(creds, scopes)
         .then((http.Client client) {
       if (trace) client = new TraceClient(client);
-      return callback(project, client).whenComplete(() => client.close());
+      return callback(project, client);
```

Comment:
Doesn't have to be now, but we should refactor this method and call it something other than `withAuthClient`, now that it doesn't manage the life-cycle of the client anymore.

Reply:
I had to refactor the code first to remove legacy dependencies to SimpleConfiguration.
Sorry, I won't have time to do it now.

Reply:
Don't worry, this was just as much a note for me to get back to this later. You've done plenty, thanks!